### ISSUE TITLE
batchnorm1d doesn't seem to be used in attention block

### ISSUE THREAD

**Issue by filipmu** [qiuqiangkong/audioset_tagging_cnn#47]
In class `AttBlock(nn.Module)` the  `__init__`  has

`self.bn_att = nn.BatchNorm1d(n_out)`

but the `forward` doesn't seem to be using it.

Also, `temperature` variable does not seem to be used.

Can these be removed without affecting the learning?
